src/manual/task-execution/worker-types.md

```diff
@@ -0,0 +1,36 @@
+---
+filename: task-execution/worker-types.md
+title: Worker Types
+order: 30
+---
+
+Worker types can serve a few different purposes within Taskcluster.
+
+Different worker types may have different system configurations. For example,
+one worker type may execute tasks in a Windows environment, while another
+executes tasks within an iOS application context. More subtly, two worker types
+might execute the same "kind" of tasks, but in instances with varying resources
+such as memory and CPU, and correspondingly varying costs.
+
+Worker types also provide a security boundary. Taskcluster's access control
+model limits what can create a task for each worker type. This can control
+access to workers with special powers, such as one that can upload binaries for
+release. It can also serve to isolate tasks with different trust levels from
```

Comment:
That's not what I mean here - I'm talking about `queue:create-task:some-provisioner/release-worker`, not about special release-specific scopes.